staging/src/k8s.io/client-go/util/cert/BUILD

```diff
@@ -21,12 +21,20 @@ go_library(
     srcs = [
         "cert.go",
         "csr.go",
+        "entropy.go",
+        "entropy_other.go",
         "io.go",
         "pem.go",
-    ],
+    ] + select({
+        "@io_bazel_rules_go//go/platform:linux_amd64": [
+            "entropy_linux.go",
```

Comment:
this didn't get checked in. I'm terrified of what it will contain

Reply:
No reason to. Only reading the integer from `/proc/sys/kernel/random/entropy_avail`.